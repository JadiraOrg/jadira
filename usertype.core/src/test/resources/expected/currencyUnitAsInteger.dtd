<!ELEMENT dataset (
    CURRENCYUNITASINTEGER*)>

<!ELEMENT CURRENCYUNITASINTEGER EMPTY>
<!ATTLIST CURRENCYUNITASINTEGER
    ID CDATA #REQUIRED
    CURRENCYUNIT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

