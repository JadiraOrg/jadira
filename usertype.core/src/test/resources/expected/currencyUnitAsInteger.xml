<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "currencyUnitAsInteger.dtd">
<dataset>
  <currencyUnitAsInteger ID="0" CURRENCYUNIT="978" NAME="test_0"/>
  <currencyUnitAsInteger ID="1" CURRENCYUNIT="840" NAME="test_1"/>
  <currencyUnitAsInteger ID="2" CURRENCYUNIT="826" NAME="test_2"/>
  <currencyUnitAsInteger ID="3" CURRENCYUNIT="682" NAME="test_3"/>
  <currencyUnitAsInteger ID="4" NAME="test_4"/>
</dataset>